homeassistant/components/sensor/magicseaweed.py

```diff
@@ -0,0 +1,212 @@
+"""
+Support for magicseaweed data from magicseaweed.com.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.magicseaweed/
+"""
+from datetime import timedelta
+import logging
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_API_KEY, CONF_NAME, CONF_MONITORED_CONDITIONS, ATTR_ATTRIBUTION)
+import homeassistant.helpers.config_validation as cv
+import homeassistant.util.dt as dt_util
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['magicseaweed==1.0.0']
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_HOURS = 'hours'
+CONF_SPOT_ID = 'spot_id'
+CONF_UNITS = 'units'
+CONF_UPDATE_INTERVAL = 'update_interval'
+
+DEFAULT_UNIT = 'us'
+DEFAULT_NAME = 'MSW'
+DEFAULT_ATTRIBUTION = "Data provided by magicseaweed.com"
+
+ICON = 'mdi:waves'
+
+API_URL = 'http://magicseaweed.com/api/{}/forecast/'
```

Comment:
Weird to include a url just for an error message. Just remove the url from the message.

Reply:
Making these requested changes now. Was this referencing the correct line?

Reply:
Ah, I see. This was left over from before I made the library.